lib/spack/spack/cmd/diff.py

```diff
@@ -57,37 +60,34 @@ def compare_specs(a, b, to_string=False, color=None):
     Arguments:
         a (spack.spec.Spec): the first spec to compare
         b (spack.spec.Spec): the second spec to compare
```

Comment:
How do I document this is either a Spec or a list of specs?

Reply:
```suggestion
        a (spack.spec.Spec or list): the first spec to compare
        b (spack.spec.Spec or list): the second spec to compare
```